torch/_dynamo/output_graph.py

```diff
@@ -670,12 +673,12 @@ def bind_symint(s: torch.SymInt, prop):
             # (this is harmless because we do remove the unused ones later)
             proxy = self.root_tracer.create_graph_input(
                 str(s0),
-                torch.SymInt,
+                type(s),
```

Comment:
Is there a reason to pass type(s) if there's always an example value? create_graph_input can always get the type from the example value.